srcpkgs/opencl-amd/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'opencl-amd'
+pkgname=opencl-amd
+version=18.50.708488
+revision=1
+create_wrksrc=yes
+hostmakedepends="wget"
+depends="ocl-icd"
+short_desc="OpenCL userspace driver as provided by the amdgpu-pro driver stack"
+maintainer="Adam Buechner <adi51@gmx.de>"
+license="custom"
+homepage="http://www.amd.com"
+archs="x86_64"
+repository=nonfree
+
+_prefix="amdgpu-pro-"
+_postfix="-ubuntu-18.04"
+_major="18.50"
+_minor="708488"
+
+distfiles="https://drivers.amd.com/drivers/linux/${_prefix}${_major}-${_minor}${_postfix}.tar.xz"
+checksum="b7b7bc9796e524e35f1f3ccbd99ba89cea383a0ed5a5357084522da0bdc09d59"
+
+XBPS_FETCH_CMD="wget --referer https://support.amd.com/en-us/kb-articles/Pages/AMDGPU-PRO-Driver-for-Linux-Release-Notes.aspx -N"
```

Comment:
Try to export [HTTP_REFERER](https://github.com/void-linux/void-packages/pull/7514#issuecomment-456050166) instead.